### PR TITLE
Upgraded all projects to EncompassSDK 18.2

### PR DESCRIPTION
**Pull request by GeorgeGomez**

### CODE REVIEW

File: GuaranteedRate.Sextant.CustomFieldComparer/packages.config [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommandLineParser" version="1.9.71" targetFramework="net452" />
-  <package id="EncompassSDK.Complete" version="18.1.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net452" />
```
Comment (johnholmer) [inline GuaranteedRate.Sextant.CustomFieldComparer/packages.config]: Did the reference to EncompassSDK.Complete get removed from this project?  I would have expected to see the project file with a hundred deleted lines for all the SDK file if that was the case?

File: GuaranteedRate.Sextant/GuaranteedRate.Sextant.csproj [hunk 2]
```diff
@@ -32,115 +32,111 @@
...
     <Reference Include="EMExport, Version=1.5.1.0, Culture=neutral, PublicKeyToken=d11ef57bba4acf91, processorArchitecture=MSIL">
-      <HintPath>..\packages\EncompassSDK.Complete.18.1.0.1\lib\net45\EMExport.dll</HintPath>
+      <HintPath>..\packages\EncompassSDK.Complete.18.2.0.2\lib\net45\EMExport.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="EMLoanUtils15, Version=1.5.1.0, Culture=neutral, PublicKeyToken=d11ef57bba4acf91, processorArchitecture=MSIL">
-      <HintPath>..\packages\EncompassSDK.Complete.18.1.0.1\lib\net45\EMLoanUtils15.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="EncompassAutomation, Version=17.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (johnholmer) [inline GuaranteedRate.Sextant/GuaranteedRate.Sextant.csproj]: Not sure why EncompassAutomation isn't referenced in the other project any longer.  That's not consistent with what the other projects did, is it required for this project?

Comment (HarryCordewener) [reply GuaranteedRate.Sextant/GuaranteedRate.Sextant.csproj]: NuGet probably detected it was not being used by this project.

File: GuaranteedRate.Sextant.CustomFieldComparer/packages.config (hunk 1 above)
Comment (HarryCordewener) [reply GuaranteedRate.Sextant.CustomFieldComparer/packages.config]: NuGet probably detected it was not being used by this project.

**Pull request merged**